documentation/service_admin/installing-kubernetes.adoc

```diff
@@ -0,0 +1,90 @@
+[[installing-openshift]]
+
+== Installing {ProductName} on Kubernetes
+
+[[prerequisites]]
+=== Prerequisites
+
+To install {ProductName}, you need to have Kubernetes installed. You can use
+https://github.com/kubernetes/minikube[minikube] if you want to install {ProductName} on your
+laptop.
+
+Since minikube runs in a virtual machine, you need to enable the ingress controller:
+
+[source,options="nowrap"]
+....
+minikube addons enable ingress
+....
+
+.Procedure 
+
+. Download one of the releases from https://github.com/EnMasseProject/enmasse/releases and unpack it.
```

Comment:
Minor point, but if we are parameterising the product name, we should also parameterise the link to releases. (Not a blocking issue on this PR though).